apps/src/applab/applab.js

```diff
@@ -663,10 +663,11 @@ Applab.init = function (config) {
 
   if (showDebugButtons || showDebugConsole) {
     debuggerUi = new JsDebuggerUi(Applab.runButtonClick);
-    extraControlRows = debuggerUi.getMarkup(studioApp.assetUrl, {
+    var extraControlRowsHtml = debuggerUi.getMarkup(studioApp.assetUrl, {
       showButtons: showDebugButtons,
       showConsole: showDebugConsole
     });
+    extraControlRows = <ProtectedStatefulDiv dangerouslySetInnerHTML={{ __html : extraControlRowsHtml }} />;
```

Comment:
Better fix in the future is probably to have debuggerUi.getMarkup give us a component instead of html.